apps/nrm/behaviours/format-answers.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+/**
+ * Capitalise the first letter or the first letter of each word (Title Case)
+ *
+ * @param {Object} text - text to be capitalised
+ * @param {bool} [all=false] - whether the first letter of each word is capitalised
+ *
+ * @returns {string} - capitalised text
+ */
+const capitaliseText = (text, all = false) => {
+  const capitalisedText = text.charAt(0).toUpperCase() + text.slice(1);
+
+  if (all) {
+    var splitString = text.toLowerCase().split(' ');
+    for (var i = 0; i < splitString.length; i++) {
+      splitString[i] = splitString[i].charAt(0).toUpperCase() + splitString[i].substring(1);
+    }
+    return splitString.join(' ');
+  }
+
+  return capitalisedText;
+
+};
+
```

Comment:
Nice
Should this and the one below be in utils folder as a separate module as pdf part could reuse?